tests/modules/home-environment/session-variables.nix

```diff
@@ -1,8 +1,18 @@
-{ config, lib, ... }:
+{ config, lib, pkgs, ... }:
 
 with lib;
 
-{
+let
+  inherit (pkgs.stdenv.hostPlatform) isLinux;
+
+  expectedConf = pkgs.substituteAll {
+    src = ./session-variables-expected.txt;
+    # the blank space below is intentional
+    exportLocaleVar = optionalString isLinux ''
+
+      export LOCALE_ARCHIVE_2_27="${pkgs.glibcLocales}/lib/locale/locale-archive"'';
```

Comment:
I've pushed a fix for this.

Reply:
Oh wait, the old formatting was actually correct. Moving the `'';` to the next line would cause the test to fail because it would generate an unwanted newline character.